### DISCUSSION TITLE
Issue with Restoring Project - Database Unresponsive

### DISCUSSION THREAD

**Discussion by princesoniversatile** [supabase/supabase#28270] [Questions]
Hello Supabase Community,

I am facing an issue while trying to restore my project "crm-mui" on the Free plan. I received the following error message:

"Something went wrong while restoring your project. Your project's data is intact, but your project is inaccessible due to the restoration failure. Please contact support for assistance."

Details:

Project Name: crm-mui
Issue: Database unresponsive and project restoration failure
Severity: Low
I have attached a screenshot of the error message for reference. Can anyone guide me on how to resolve this issue? Any help would be greatly appreciated.

![Screenshot 2024-07-30 114848](https://github.com/user-attachments/assets/fb3a5efb-961f-4079-a1a9-5e30929e86db)
![Screenshot 2024-07-30 114914](https://github.com/user-attachments/assets/fa8fd151-43dc-409d-a9f1-9308e22bcee0)
![Screenshot 2024-07-30 115000](https://github.com/user-attachments/assets/68b9a4cb-0711-4d4e-afec-fc2a72e6e9ea)

**Comment by GaryAustin1** [comment]
You to file a support request.  No one outside of Supabase can  help on this.

**Reply by princesoniversatile** [reply]
I did and waiting for resolution.

**Reply by princesoniversatile** [reply]
May I know why this thing happening .

**Reply by GaryAustin1** [reply]
I've never seen them give a specific reason, and it could be several different things could cause it.  You might ask support and see if they say exactly what was wrong with your instance.

**Reply by ensticks** [reply]
Supporter says "Free plan will receive no guaranteed support response" 😆

**Reply by TechBoyy6** [reply]
happend with me too 😭, i tho dont have the backup of the data either. Talking with their support AI, it says while restoration the confirmation might not be given, which i feel is the case for me, bcaz i started the resume process and then i closed the tab 🤦🏻‍♂️

**Reply by faaiz99** [reply]
same issue

**Reply by princesoniversatile** [reply]
My Issue Resolved Thanks To Supa base Team.

**Reply by TechBoyy6** [reply]
was there something u had to do or?

**Reply by tanmay-shikha-labs** [reply]
Facing the same issue. @princesoniversatile @faaiz99 @TechBoyy6 did this issue get resolved for you too? How long did this take to resolve?

**Reply by faaiz99** [reply]
Hi @tanmay-shikha-labs, 

I reached to their support from within the dashboard. It took maybe max 2 weeks and it was resolved then. 


I hope this helps

Faaiz

**Reply by tanmay-shikha-labs** [reply]
Thanks a lot @faaiz99 !

**Comment by LoordhuJeyakumar** [comment]
Hit something similar a while back a restore that looked fine but wasn't, and no way to know until it actually mattered. I'm building a small tool that backs up Supabase projects and then actually restores them on a schedule to prove the backup is good, instead of just assuming a scheduled backup means you're safe. Still early. If that's something you'd want to try once it's ready, happy to add you to an early access list  no pitch, just trying to figure out if this is worth building for people who've actually hit this.
